study.md

```diff
@@ -43,7 +50,9 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
```

Comment:
This does not answer the question.

`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns each digit of π on an iteration has does not have a finite length.